init.sh

```diff
@@ -0,0 +1,62 @@
+echo "Starting Simulation Central Controller"
+python sim_controller/flock_controller/main.py &
+echo "Central Controller started successfully!"
+
+## Wait for 2 seconds.
+/bin/sleep 4
```

Comment:
Make all sleep to 8 seconds, there may be synchronisation errors if the values are small.